control-plane/tests/test_issue812_zammad_live_pilot_boundary_docs.py

```
@@ -0,0 +1,94 @@
+            "When Zammad is unavailable or credentials fail custody validation, operators may continue AegisOps case review, approval, execution, and reconciliation from AegisOps records.",
+            "Operators must not infer ticket existence, approval, execution, reconciliation, closure, or customer notification from a missing, stale, unreachable, or mismatched Zammad record.",
+            "No failed Zammad write, stale read, timeout, proxy failure, auth failure, or degraded ticket payload may create an orphan AegisOps authority record or mark an AegisOps lifecycle step complete.",
+        ):
+            self.assertIn(term, text)
+
+    def test_zammad_live_pilot_verifier_is_present_and_passes(self) -> None:
+        verifier = REPO_ROOT / "scripts" / "verify-zammad-live-pilot-boundary.sh"
+
+        self.assertTrue(verifier.exists(), f"expected verifier at {verifier}")
+
+        result = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)